x-pack/filebeat/tests/system/test_httpjson.py

```diff
@@ -0,0 +1,611 @@
+import json
+import jinja2
+import os
+import random
+import sys
+
+from datetime import datetime
+from flask import Flask, jsonify, request
+from multiprocessing import Process
+
+sys.path.append(os.path.join(os.path.dirname(__file__),
+                             '../../../../filebeat/tests/system'))
+
+from filebeat import BaseTest
+
+
+class Test(BaseTest):
```

Comment:
I had some issues trying to spawn the cursor input manager in unit tests, that was the reason why I moved the integration tests to python. Since your following comments made a lot of sense, and I implemented the stateless input instead as a first step, I moved them back to go now, since the issues I faced with the cursor input seemed to not apply with that one.